saline_lakes_mapping_script.R

```diff
@@ -0,0 +1,166 @@
+library(ggplot2)
```

Comment:
Is this the same as `mapping/saline_lakes_mapping_script.R`?

Reply:
If not, needs a new name and to be placed somewhere that informs of its' purpose. Otherwise, delete